Dockerfile

```diff
@@ -117,22 +117,36 @@ COPY gunicorn.conf.py .
 # setup docker-specific things
 # Use mounts to avoid copying installer files into the image
 # These change sometimes, but rarely
-WORKDIR /usr/src/paperless/src/docker/
+ARG DOCKER_SRC=/usr/src/paperless/src/docker/
+WORKDIR ${DOCKER_SRC}
+
+COPY [ \
+	"docker/imagemagick-policy.xml", \
+	"docker/supervisord.conf", \
+	"docker/docker-entrypoint.sh", \
+	"docker/docker-prepare.sh", \
+	"docker/paperless_cmd.sh", \
+	"docker/wait-for-redis.py", \
+	"docker/management_script.sh", \
+	"docker/install_management_commands.sh", \
+	"${DOCKER_SRC}" \
+]
 
-RUN --mount=type=bind,readwrite,source=docker,target=./ \
-  set -eux \
+RUN set -eux \
```

Comment:
Presumably this is the change made for SELinux support.  It feels like a lot of change to support what maybe isn't so common?  What's the issue with adding the `z` mount option instead?

Reply:
Why make changes to the host filesystem when you don't have to? As I run mostly selinux systems, I would not appreciate it being done without my permission.

Reply:
In the end, it's probably just a few kbs of files left in the image, which is sizeable enough that's basically nothing.

Reply:
Adding `rm install_management_commands.sh management_script.sh` at the end of the `RUN` block would accomplish basically the same thing, as those are the only files that would be unused.

Reply:
Once files have been added with COPY, their size always exists at that layer of the image, even if `mv`ed or `rm`ed, thanks to the overlay file system.  But it's just text files here, so not really an issue